client/app/templates/Ndr/Formatted/_submission_declaration.html.twig

```diff
@@ -1,4 +1,11 @@
 {% set translationDomain = "ndr-declaration" %}
+{% if app.user.isLayDeputy %}
```

Comment:
Do we need these if statements in 4 different places? Is there no higher level twig file where these values can be set?